preconfigured-packages.json

```diff
@@ -2,21 +2,21 @@
   "packages": [
     {
       "name": "ffmpeg",
-      "tag": "ffmpeg",
       "mac": {
-        "package": "ffmpeg[avcodec,avdevice,avfilter,avformat,avresample,postproc,swscale,swresample,mp3lame,sdl2,zlib]",
+        "package": "ffmpeg[avcodec,avdevice,avfilter,avformat,avresample,swscale,swresample,mp3lame]",
         "linkType": "dynamic",
-        "buildType": "release"
+        "buildType": "release",
+        "vcpkgHash": "0695a7d231daf2b72c2851faea45c46c9ec5bf1a"
```

Comment:
Why does ffmpeg need a specific hash of vcpkg?

Reply:
Oh, I just need to merge this branch and then I can remove that.  I didn't see you already approved this last week: https://github.com/TechSmith/vcpkg/pull/10.

Reply:
Fixed: 7421dea3070d803310f614feaeec4aaa271d2fd8